tests/registry/jsr/@std/url/0.220.1/join.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
 // This module is browser compatible.
 
-import { join as posixJoin } from "jsr:/@std/path@^0.220.1/posix/join";
```

Comment:
Why this change?

Reply:
I didn't check but I'm pretty sure this would bypass the faulty case, where tsc is using the form without the leading slash and that fails to match any import map key.